updating UI from source project

zeedevmobile opened the pull request:
once everything is updated i still need to go back ad polish a little, not able to test it currently

tir38 commented on app/build.gradle:
```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.application'
-    compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    compileSdkVersion 27
+    buildToolsVersion '27.0.3'
```
Gradle didn't like this?

zeedevmobile replied:
the was probably from when i had to manually update the module settings for some reason

tir38 commented on app/src/main/java/io/orionlabs/android/interview/GroupListFragment.java:
```diff
@@ -26,16 +33,16 @@ public void onAttach(Context context) {
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_group_list, container, false);
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.view_drawer_group, container, false);
```
wanna rename to  `view_group_list` ?

tir38 commented on app/src/main/java/io/orionlabs/android/interview/GroupListFragment.java:
```diff
@@ -3,20 +3,27 @@
+public class GroupListFragment extends Fragment implements GroupListContract.PolicyDelegate {
 
     @Inject
     GroupListPresenter presenter;
```
Do you wanna do the same thing here and have this depend on the interface?

tir38 commented on app/src/main/res/layout/list_item_group.xml:
```diff
@@ -53,27 +53,26 @@
-        <!--app:layout_constraintEnd_toStartOf="@+id/guideline_list_item_group_vertical_begin"-->
-        <!--app:layout_constraintStart_toStartOf="parent"-->
-        <!--app:layout_constraintTop_toTopOf="parent"/>-->
+    <com.onbeep.obiwan.component.SelectableImageView
```
did you pull this back into the project?

zeedevmobile commented:
added a few updates to master:
1. Switched to latest MVP pattern
2. Cleaned up a couple minor UI issues
3. Added a couple more groups to make the list look better.
